src/Hl7.Fhir.Support.Poco.Tests/NewPocoSerializers/FhirJsonSerializationTests.cs

```diff
@@ -41,7 +41,8 @@ public void RoundtripEdgeCases()
         {
             var (poco, expected) = getEdgecases();
 
-            var options = BaseOptions.Pretty();
+            var options = new JsonSerializerOptions().ForFhir(typeof(TestPatient).Assembly).Pretty();
```

Comment:
This is exactly why this could be considered a breaking change, as we discussed on our call. Still, the benefits for the use in service initialization outweigh the risk I think. And I guess many people even assume we return the same instance (but then modified). But we cannot be sure.

Reply:
I'll add something to the release notes